pkg/svcgen/providers.go

```diff
@@ -0,0 +1,25 @@
+package svcgen
+
+import (
+	"github.com/google/go-cloud/wire"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+	"github.com/izumin5210/grapi/pkg/protoc"
+	"github.com/izumin5210/grapi/pkg/svcgen/params"
+)
+
+func ProvideParamsBuilder(rootDir cli.RootDir, protocCfg *protoc.Config, grapiCfg *grapicmd.Config) params.Builder {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideParamsBuilder should have comment or be unexported